libguard/guard_exception.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <exception>
+#include <sstream>
+#include <string>
+
+namespace openpower
+{
+namespace guard
+{
+namespace exception
+{
+// TODO:Issue #3325, to avoid multiple exception classes
```

Comment:
should be creating issue in guard repository.  
TODO #3325 is useless unless you specify the repository name .

Reply:
Created issue in guard repo. 
https://github.com/open-power/guard/issues/3